### ISSUE TITLE
Generating black images and model not getting trained

### ISSUE THREAD

**Issue by musafirsafwan** [lukedeo/keras-acgan#4]
Same issue as #1 

Downloaded the code and ran it without any editing.
The saved images after each epoch all are completely black. The numbers reported after each epoch are almost same. Issue is not solved even after updating to the latest versions of tensorflow and keras.
Versions:
tensorflow-gpu=0.12.1
Keras=1.2.0

**Comment by tpbarron** [comment]
I have tried to run the code and am experiencing the same problem. After 10, 20, and 30 batches the images look like this:

![plot_epoch_000_generated](https://cloud.githubusercontent.com/assets/763829/22048021/d60600ee-dce7-11e6-8678-7e31c25ef5c7.png)

![plot_epoch_001_generated](https://cloud.githubusercontent.com/assets/763829/22048025/daaf3ee4-dce7-11e6-8318-21cbba90dda7.png)

![plot_epoch_002_generated](https://cloud.githubusercontent.com/assets/763829/22048027/def42276-dce7-11e6-818f-5c005cde5a93.png)

**Comment by maayanfa** [comment]
Also for me with latest versions for tensorflow-gpu and keras

**Comment by newikov** [comment]
Also for me :((( help guys!

**Comment by dremovd** [comment]
+1

**Comment by dremovd** [comment]
@newikov @maayanfa @tpbarron @musafirsafwan 
Managed to reproduce original results by decreasing learning rates, see #5

**Comment by sautin1** [comment]
Problem is still actual for me (even after decreasing the lr).
Win10, Python 3.5.2 | Anaconda 4.2.0 (64-bit)
tensorflow-gpu==0.12.1,
Keras==1.2.2

**Comment by xhchrn** [comment]
dremoved's fix is right. I decrease learning rate from 0.0002 to 0.0001, and the model works.

**Comment by xuewenyuan** [comment]
I have the same problem, which are solved when I decrease learning rate to 0.00002.

**Comment by showskyzt** [comment]
Thanks to #4, I changed the lr from 0.0002 to 0.00005, and the model works!
